kmip/services/server/session.py

```diff
@@ -97,6 +97,8 @@ def run(self):
         """
         self._logger.info("Starting session: {0}".format(self.name))
 
+        self._connection.do_handshake()
```

Comment:
This should be wrapped in a try/except/else block in case the handshake fails. The except block should look something like L107-109 so that we log the handshake failure. The else block should just contain the while-loop at L102. If the handshake fails, we log the failure, skip the while-loop, and go straight to cleaning up the connection and closing out the session.

Reply:
not related to this PR, @PeterHamilton I also packaged some pretty trivial Docker image, are you interested in that? I can PR it

Reply:
thanks, updated